README.md

```diff
@@ -119,6 +119,18 @@ public class LoggerDelegate implements JavaDelegate {
 
 and then reference it the process model with the expression`${loggerDelegate}`.
 
+## Blocking Operations 
```

Comment:
Heading: "Executing Blocking Operations on I/O Thread Pool"

->I/O threads
-> possible deadlocks
-> Camunda
-> 7.14 -> current
-> only link once to OCI
-> which interacts with the process engine
-> By default, 
-> for this blocking operation.
-> for more information -> link annotation.

@Hayvon : let's have a short Zoom Meeting tomorrow to fix the documentation together.